solver/llbsolver/ops/file.go

```diff
@@ -421,7 +421,6 @@ func (s *FileOpSolver) getInput(ctx context.Context, idx int, inputs []fileoptyp
 					if cerr == nil {
 						outputRes[idx-len(inputs)] = worker.NewWorkerRefResult(ref.(cache.ImmutableRef), s.w)
 					}
-					inpMount.Release(context.TODO())
```

Comment:
@hinshun I don't understand why in line 413 it is not `input.(cache.ImmutableRef).Clone()`, but if I add it `TestClientGatewayExecFileActionError` does not release everything and fails. Without it, I don't see any reports of double release.